src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import Home from './pages/Home';
+import Detail from './pages/Detail';
+import Error from './pages/Error';
+import Navigation from './components/Navigation';
+import { updateInitialData } from './Redux/App/App';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(updateInitialData());
+  }, []);
+  return (
+    <BrowserRouter>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/detail" element={<Detail />} />
+        <Route path="/error" element={<Error />} />
```

Comment:
- [ ] To follow the design more closely I would suggest adding these buttons that were presented in the design:
![Screenshot from 2022-05-18 20-47-48](https://user-images.githubusercontent.com/62705947/169131177-bea7bd74-a04d-4644-a694-2ed748df75f5.png)

Reply:
### Will do @GzimAsani . Thank you.